packages/api/src/routes/internal.ts

```diff
@@ -267,4 +279,46 @@ router.post(
   })
 );
 
+/**
+ * GET /internal/cx-hie-status
+ *
+ * Retrieves the customer status of enabled HIEs via the Feature Flags.
+ *
+ * @param req.query.cxId - The cutomer's ID.
+ */
+router.get(
+  "/cx-hie-status",
+  requestLogger,
+  asyncHandler(async (req: Request, res: Response) => {
+    const cxId = getUUIDFrom("query", req, "cxId").orFail();
+    const result = await getHieEnabledFFStatus(cxId);
+    return res.status(httpStatus.OK).json(result);
+  })
+);
```

Comment:
Could we just do a single endpoint that returns all the FF status for a given customer? We'll need to expose the other ones at some point, and it should be straightforward to just return all cx-based ones in one endpoint instead of building diff endpoints for diff use cases.

Reply:
Done.

Reply:
It seems you just missed updating the endpoint's path, maybe replace `hie` by `ff` or `feature-flag`?

Reply:
Good call. Done.